operator/pkg/name/name.go

```diff
@@ -49,8 +49,15 @@ const (
 	IngressComponentName ComponentName = "IngressGateways"
 	EgressComponentName  ComponentName = "EgressGateways"
 
-	// Addon components
-	AddonComponentName ComponentName = "Addon"
+	// Addon root component
+	AddonComponentName ComponentName = "AddonComponents"
```

Comment:
still seems to be here. anyway, since the patch is in master you can cherry pick that one separately afterwards if you prefer, this is just a heads up. lmk and i will approve.